src/.vuepress/theme/Search.vue

```diff
@@ -12,7 +12,7 @@
           type="text"
           class="md-search__input"
           name="query"
-          placeholder="Search (s)"
+          placeholder="Search (es)"
```

Comment:
Why this change ? If I remember well, the `s` key is the shortcut to open the Algolia search

Reply:
I thought it was the plural with a mistake, my bad !